src/NServiceBus.Core/DelayedDelivery/TimeoutManager/TimeoutManagerRecoverabilityPolicy.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace NServiceBus
+{
+    using Transport;
+
+    class TimeoutManagerRecoverabilityPolicy : IRecoverabilityPolicy
+    {
+        public TimeoutManagerRecoverabilityPolicy()
+        {
+            RaiseNotifications = false;
+        }
+
+        public RecoverabilityAction Invoke(ErrorContext errorContext)
+        {
+            if (errorContext.NumberOfDeliveryAttempts <= MaxNumberOfFailedRetries)
+            {
+                return RecoverabilityAction.ImmediateRetry();
+            }
+
+            return RecoverabilityAction.MoveToError();
+        }
+
+        public bool RaiseNotifications { get; }
```

Comment:
I'm OK with the rename. We were also not super stoked by the approach. But we didn't want to open up the pandora's box